tests/x11/vnc_two_passwords.pm

```diff
@@ -132,15 +135,19 @@ sub run {
         # eg.
         #     xev -display $display -root | tee /tmp/xev_log ; echo xev-finished >/dev/$serialdev
         # will not work
-        # Parentheses are needed to not populate trap to following commands
-        enter_cmd "(trap 'echo xev-finished >/dev/$serialdev' SIGINT; xev -display $display -root | tee /tmp/xev_log) ";
+        # Parentheses are needed to not populate trap to following commands. Add delay time on aarch64 because xev takes longer to finish or even hangs
+        my $delay_time = (is_aarch64 && is_sle) ? 60 : 1;
+        enter_cmd "(trap 'echo xev-finished >/dev/$serialdev' SIGINT; xev -display $display -root | tee /tmp/xev_log); sleep $delay_time; ";
 
         # Repeat with RO/RW password
         generate_vnc_events $opt->{pw};
 
-        # Close xev
+        # Close xev, re-try if needed on aarch64, increase timeout on aarch64 for wait_serial, see poo#119416
+        my $timeout = (is_aarch64 && is_sle) ? 90 : 30;
         send_key 'ctrl-c';
-        wait_serial('xev-finished') || die 'xev not finished';
+        # give a second chance to quit xev
+        send_key 'ctrl-c' if (is_sle && is_aarch64) && ((check_screen 'need-to-close-again', $timeout) || (check_screen 'need-to-close-again-extra1', $timeout));
+        wait_serial('xev-finished', $timeout) || die 'performance issue on aarch64, see poo#120282 for more details';
```

Comment:
Die messages are for all platforms, not only for aarch64, correct?

Reply:
I checked, other plattforms don't have the issue at all, won't see this message, but this is good point, let me think how to limit this to aarch64 only, thanks!